lib/st25r3911b/st25r3911b_common.c

```diff
@@ -529,21 +554,59 @@ int st25r3911b_field_on(u8_t collision_threshold, u8_t peer_threshold,
 int st25r3911b_technology_led_set(u32_t led, bool on)
 {
 	int err;
+	gpio_pin_t pin;
+
+	switch (led) {
+	case ST25R3911B_NFCA_LED:
+		if (DT_NODE_HAS_PROP(ST25R3911B_NODE, led_nfca_gpios)) {
+			pin = NFCA_LED_PIN;
+			gpio_dev = device_get_binding(NFCA_LED_PORT);
+			if (!gpio_dev) {
+				LOG_ERR("GPIO device binding error.");
+				return -ENXIO;
+			}
```

Comment:
This part is common for all cases. Why not place it after the switch (and leave here only `name = NFCA_LED_PORT;`)?

Reply:
Ok. Changed